lld/ELF/Symbols.cpp

```diff
@@ -146,7 +146,7 @@ static uint64_t getSymVA(const Symbol &sym, int64_t addend) {
   case Symbol::SharedKind:
   case Symbol::UndefinedKind:
     return 0;
-  case Symbol::LazyObjectKind:
+  case Symbol::LazySymbolKind:
```

Comment:
I think this can also just be `LazyKind` since its matches the other kinds.

Reply:
Thanks for the suggestion!